File: day1.py
with open("/tmp/aoc/input.1") as input:
    loads = [n for n in reversed(sorted([sum([int(a) for a in n.split("\n") if a != ""]) for n in input.read().split("\n\n")]))]
    print("Task 1: ",loads[0])
    print("Task 2: ",sum(loads[0:3]))

File: Day1.java
import java.io.IOException;
import java.nio.file.Path;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

import static java.lang.Integer.parseInt;
import static java.nio.file.Files.readAllLines;

public class Day1 {
    public static void main(String[] args) throws IOException {
        List<Integer> sums = new ArrayList<>();
        int sum = 0;
        for (var line: readAllLines(Path.of("/tmp/aoc/input.1"))) {
            if ("".equals(line)) {
                sums.add(sum);
                sum = 0;
            } else {
                sum += parseInt(line);
            }
        }
        if (sum > 0) sums.add(sum);
        sums.sort(Collections.reverseOrder());
        System.out.println("Task 1: "+sums.get(0));
        System.out.println("Task 2: "+sums.subList(0,3).stream().reduce(0, Integer::sum));
    }
}
